src/status_im/ui/screens/network_settings/events.cljs

```diff
@@ -31,19 +31,21 @@
 (defn network-with-upstream-rpc? [networks network]
   (get-in networks [network :raw-config :UpstreamConfig :Enabled]))
 
-(defn connect-network [cofx [_ network]]
-  (merge (accounts-events/account-update cofx {:network network})
-         {:close-application nil}))
-
-(handlers/register-handler-fx ::save-network connect-network)
+(handlers/register-handler-fx
+ ::save-network
+ (fn [{:keys [db now]} [_ network]]
+   (-> (accounts-events/account-update {:db db} {:network      network
```

Comment:
It could be just
```clojure
(accounts-events/account-update
 {:db                db
  :close-application nil}
 {:network      network
  :last-updated now})
```